test.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 source settings.sh
+DEVICE=ltsptest01
 
 help() {
     cat <<-_EOF
```

Comment:
why there is - (hyphen)?

Reply:
It disables leading tabs (for proper code indentation):
https://stackoverflow.com/questions/2953081/how-can-i-write-a-heredoc-to-a-file-in-bash-script
In this case maybe it is not needed.